src/resources/authoring/gv2.json

```diff
@@ -54,7 +54,7 @@
       "targetDrakes": [{},{}],
       "userChangeableGenes": "wings, forelimbs, hindlimbs"
     },
-    "eggSortingWings": {
+    "basicEggSortingWings": {
```

Comment:
Have these naming conventions been run by Trudi? I believe there's a PT story to that effect.